app/bank/domain/user.py

```diff
@@ -0,0 +1,17 @@
+import uuid
+from django.db import models
+
+
+class User(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    name = models.CharField(max_length=50)
+    surname = models.CharField(max_length=50)
+    age = models.IntegerField()
```

Comment:
do not use age, use born date

(To avoid recalculate this every day)

Reply:
Store the born date and create a public function that returns the age based on the born date ( If needed in the future)